src/typeDefinitions/default.types.d.ts

```diff
@@ -1,3 +1,8 @@
 export interface Env {
```

Comment:
Any reason of starting Env with uppercase E

Reply:
nope, since i am passing env to a function with data type as Env so kept first letter capital as a distinction.

Reply:
please change it from uppercase and rename Env.